### PR TITLE
Added main classes to use

### PR DESCRIPTION
**Pull request by Mathreyu**
- NavDrawer stuff still missing.
- Waiting for UI to add functionality to adding new TODO

### CODE REVIEW

File: todoapp/TODOapp/.idea/misc.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
     <ConfirmationsSetting value="0" id="Add" />
     <ConfirmationsSetting value="0" id="Remove" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" default="true" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (jmsalcido) [inline todoapp/TODOapp/.idea/misc.xml]: just add `.idea` folder over .gitignore and delete this folder once to remove it from the repository.

File: todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/Activities/TODOContent.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.nearsoft.androidschool.todoapp.Activities;
+
```
Comment (jmsalcido) [inline todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/Activities/TODOContent.java]: use lowercase for packages

**Comment by jmsalcido** [comment]
I prefer Todo*.java over TODO*.java, but that's your choice.

File: todoapp/TODOapp/.idea/misc.xml (hunk 1 above)
Comment (Mathreyu) [reply todoapp/TODOapp/.idea/misc.xml]: will do ;D

File: todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/Activities/TODOContent.java (hunk 2 above)
Comment (Mathreyu) [reply todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/Activities/TODOContent.java]: done :)

**Pull request merged**